### DISCUSSION TITLE
Update of French language file

### DISCUSSION THREAD

**Discussion by patrickus33** [templaza/astroid-framework#859] [General]
I've send the update version of french ini file to templaza .
[fr-FR.astroid.ini.zip](https://github.com/user-attachments/files/17600312/fr-FR.astroid.ini.zip)

**Comment by sonvnn** [comment]
@Joomlaplates My friend! Can you help me verify and update the language branch? Thank you very much :)

**Reply by patrickus33** [reply]
Hi @sonvnn 
Here is the French translation based on version 3.18 !
[fr-FR.astroid.ini.zip](https://github.com/user-attachments/files/17866532/fr-FR.astroid.ini.zip)

**Reply by sonvnn** [reply]
Thank you very much :)

**Reply by patrickus33** [reply]
You're welcome !

Let me know when you add lines to the language file.
Maybe you could put this on Tansifex or Crowdin, for example.
It would be easier to manage for translators, who would be able to see directly the additions to be translated.

**Reply by patrickus33** [reply]
You're welcome !

Let me know when you add lines to the language file.
Maybe you could put this on Tansifex or Crowdin, for example.
It would be easier to manage for translators, who would be able to see directly the additions to be translated.

**Reply by joomlaplates-de** [reply]
Hello, I just upload the new french file.
Merci
